### PR TITLE
ASRAgent: Update interface v1.4

### PR DESCRIPTION
**Pull request by JeanTracker**
ASRAgent supports EPD's timeout length adjustment function to support
long conversations.

Signed-off-by: JeanTracker <hyojoong.kim.jean@gmail.com>

### CODE REVIEW

File: src/core/speech_recognizer.cc [hunk 1]
```diff
@@ -298,6 +301,22 @@ void SpeechRecognizer::stopListening()
     AudioInputProcessor::stop();
 }
 
+void SpeechRecognizer::setEPDAttribute(const EpdAttribute& attribute)
+{
+    epd_timeout = attribute.epd_timeout;
```
Comment (webispy) [inline src/core/speech_recognizer.cc:306]: Please check time unit(secs / milliseconds)
* EPD fields in the ASR directive uses milliseconds.
* speech_recognizer uses secs. (core/speech_recognizer.cc line 34~36)

Comment (JeanTracker) [reply src/core/speech_recognizer.cc:306]: I fixed it. Thanks :)

File: include/clientkit/speech_recognizer_interface.hh [hunk 2]
```diff
@@ -102,6 +103,18 @@ public:
      */
     virtual void stopListening() = 0;
 
+    /**
+     * @brief Set EPD attribute
+     * @param[in] attribute epd attribute
+     */
+    virtual void setEPDAttribute(const EpdAttribute& attribute) = 0;
```
Comment (kimhyungrok) [inline include/clientkit/speech_recognizer_interface.hh:110]: How about to unify case about "EPD? Epd" between "setEPDAttribute()" and "EpdAttribute" including "getEPDAttribute() below?

Comment (JeanTracker) [reply include/clientkit/speech_recognizer_interface.hh]: Ok. I fixed it.

**Review by kimhyungrok — APPROVED** [review]

**Review by webispy — APPROVED** [review]

**Pull request merged**